internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -50,6 +50,26 @@ username: %s
 password: %s
 `
 
+const clusterDetails = `
+[Set up your Atlas cluster]
+`
+
+const databaseUserDetails = `
+[Set up your database access details]
+`
+
+const accessListDetails = `
+[Set up your Access list details]
```

Comment:
[nit] a user may not understand this unless we mention network
```suggestion
[Set up your network access list details]
```